tests/functional/spec/ukis/routing/which-years.spec.js

```diff
@@ -0,0 +1,196 @@
+import chai from 'chai'
+import {openQuestionnaire} from '../../../helpers'
+
+import GeographicMarkets from '../../../pages/surveys/ukis/geographic-markets.page.js'
```

Comment:
We should look at which imports we actually have to import and which can be deleted. I wonder if atom or intellij has a optimise imports for node?